test-e2e/test_e2e_module.py

```diff
@@ -13,12 +13,12 @@
 @pytest.fixture(scope='session', autouse=True)
```

Comment:
While here I also decided to update the comment (review comment from @jgehrcke a while back) and reduce the amount of logs in the build log, as was previously done in DC/OS Enterprise by suppressing some logs from `sarge`.